### PR TITLE
Fix Travis CI builds

### LINKED ISSUE #49: The build started failing on master
The crypto-square exercise has started failing on Travis CI. I can't see that it has changed, but suddenly started failing.

@celavek would you have a moment to take a look at this?

### PR DESCRIPTION
**Pull request by ohdatboi**
Fixes #49.

The build failed because of the deprecation warnings. They aren't a big deal unless `
buildRequirements "disallowDeprecations"` is specified in the `dub.sdl` and this is exactly what happened.

I don't know why such option is in `dub.sdl`, because a lot of stuff in the D's standard library get deprecated and builds will continue failing for "no reason".

### CODE REVIEW

**Comment by kytrinyx** [comment]
> I don't know why such option is in dub.sdl, because a lot of stuff in the D's standard library get deprecated and builds will continue failing for "no reason".

That's a good point.

Maybe instead of having the build fail, we could add something in the README to suggest that people check for deprecation warnings since the project moves quickly, and we'd like to keep up where possible.

**Comment by kytrinyx** [comment]
Thanks for fixing this!

File: exercises/crypto-square/.dub/dub.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (petertseng) [inline exercises/crypto-square/.dub/dub.json]: excuse me for barging in: is this file necessary to have in this repo? what does it do? sorry for my ignorance

**Comment by petertseng** [comment]
> I don't know why such option is in `dub.sdl`

Hi, even though you say you don't know, it might not be the case that you are also asking "someone please explain". I think I know the answer, so I can explain, but I'm sorry if you didn't want me to explain!

I saw that in https://github.com/exercism/d/commit/4af61f8b226868b7c9c6e349f774769337c65451#diff-40736f68964f825efc4f1d54bfc3ab51 the recommended command to be run was `dmd example.d -de -w -unittest` Ah, I see that includes the `-de` flag, which makes deprecations errors. So this option was preserved in #30.

So:

* Why is it in the `dub.sdl`? Because it was in the original instructions.
* Why was it in the original instructions? Don't know.
* Given the two above answers, does it need to stay in `dub.sdl`? No, if current maintainers agree that removing it is best.

File: exercises/crypto-square/.dub/dub.json (hunk 1 above)
Comment (ohdatboi) [reply exercises/crypto-square/.dub/dub.json]: Oops, it shouldn't be there. I've fixed it in #55
